### DISCUSSION TITLE
Wil all the old examples reappear in the new project manager?

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#146] [Q&A] [answered]
Bacause they are nice learning material

**Comment by renaudpawlak** [comment, marked as answer]
I have re-published some of the examples (see the project manager). Not all of them so far, but there will be more eventually.

**Comment by renaudpawlak** [comment]
Note: I have just finished a very ambitious refactoring to use IndexDB instead of LocalStorage in the browser. This should make dLite much more scalable. However, you might encounter some instabilities :/ It will eventually get all solved !!!
